src/components/ChatLog.js

```diff
@@ -2,29 +2,37 @@ import React from 'react';
 import './ChatLog.css';
 import './ChatEntry.css';
 import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
 
 const ChatLog = (props) => {
   
   const chatComponents =  props.entries.map(entry => {
     return (
-      <div className="chat-log" key={entry.id}>
         <ChatEntry 
-        id={entry.id} 
+        key={entry.id}
+        id={entry.id}
         sender={entry.sender}
         body={entry.body}
         timeStamp={entry.timeStamp}
         liked={entry.liked}
-        increaseLikes={props.increaseLikes}
+        handleLikeToggle={props.handleLikeToggle}
         />
-
-    </div>
      );
     });
     return (
-      <div>
+      <div className="chat-log">
         {chatComponents}
       </div>
     )
 };
 
+ChatLog.propTypes = {
+  key: PropTypes.number.isRequired,
+  id: PropTypes.number.isRequired,
+  body: PropTypes.string.isRequired,
+  timeStamp: PropTypes.string.isRequired,
+  liked: PropTypes.bool.isRequired,
+};
```

Comment:
The prop types for ChatLog should reflect the attributes we're passing in. This would look more like
```js
ChatLog.propTypes = {
  entries: PropTypes.arrayOf(
    PropTypes.shape({
      id: PropTypes.number.isRequired,
      sender: PropTypes.string.isRequired,
      body: PropTypes.string.isRequired,
      timeStamp: PropTypes.string.isRequired,
      liked: PropTypes.bool.isRequired,
    })
  ).isRequired,
  handleLikeToggle: PropTypes.func,
};
```
We're not passing chat data fields invidividually. Rather, we pass an array of them (called `entries`). We can use the `shape` prop type helper to describe what the objects within the array look like (describing the message object data). Other than `entries`, we're also passing the callback function, which as for ChatEntry, can be left as not required to avoid the test warnings.